mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -310,6 +310,13 @@ void ControllerAction::runImpl(GoalHandle &goal_handle, AbstractControllerExecut
         goal_handle.setAborted(result, result.message);
         break;
 
+      case AbstractControllerExecution::ROBOT_DISABLED:
+        ROS_FATAL_STREAM_NAMED(name_, "Robot stopped and ignored velocity commands!");
+        controller_active = false;
+        fillExePathResult(mbf_msgs::ExePathResult::STOPPED,"Robot disabled and ignored velocity commands: ",result);
```

Comment:
F: `STOPPED` means the execution of the controllers was forcibly stopped (e.g. because patient ran out) -- this doesn't apply here. A better status for this case is probably `ROBOT_STUCK`